packages/core/contracts/optimistic-assertor/implementation/sovereign-security-manager/OwnerUnplugSovereignSecurityManager.sol

```diff
@@ -23,12 +23,12 @@ contract OwnerUnplugSovereignSecurityManager is BaseSovereignSecurityManager, Ow
         arbitrationResolutions[requestId] = ArbitrationResolution(true, arbitrationResolution);
     }
 
-    function setUnplugUnplugFromDvm(bool value) public onlyOwner {
-        unplugFromDvm = value;
+    function setDvmAsOracle(bool value) public onlyOwner {
+        disregardDvm = value;
```

Comment:
Naming `setDvmAsOracle` implies that if you pass true then expect use UMA DVM as Oracle while here `disregardDvm` would get set to true